docs/manifest/introduction.rst

```diff
@@ -38,8 +38,18 @@ and runtime specific settings.
     manifest filename with the extension `.py`. E.g. the script filename for a manifest
     with filename of ``job.tem`` is ``job.py`` in the same directory as the manifest file.
 
+.. note::
+    In case the associated script file is in a different location from the manifest you can use a symlink
+    to refer to the job.py. By default, the job will be executed in the directory where the manifest file is located.
+    To change the execution directory and use the symlink target directory to execute the script you need
+    to pass --follow-symlink.
+
+.. code-block:: shell
 
+    $ pyats run manifest job.tem --follow-symlink
 
+    $ job.py     # is a symlink for the associated script
```

Comment:
Please show symlink from the filesystem, e.g. 
```
ls -l job.py
lrwxr-xr-x  1 admin  wheel  20 Nov 29 11:36 job.py -> /tmp/job/path/job.py
```

This job will be executed in the `/tmp/job/path` directory.

Reply:
> Please show symlink from the filesystem, e.g.
> 
> ```
> ls -l job.py
> lrwxr-xr-x  1 admin  wheel  20 Nov 29 11:36 job.py -> /tmp/job/path/job.py
> ```
> 
> This job will be executed in the `/tmp/job/path` directory.

@omehrabi
Did you see this note from a reviewer?